lib/iris/fileformats/name_loaders.py

```diff
@@ -78,11 +78,11 @@ def read_header(file_handle):
     header = {}
     header['NAME Version'] = next(file_handle).strip()
     for line in file_handle:
-        words = line.split(':', 1)
+        words = line.split(b':', 1)
         if len(words) != 2:
             break
         key, value = [word.strip() for word in words]
-        header[key] = value
+        header[key.decode()] = value
```

Comment:
It applies to most/all of the changes to the NAME loader, but this is a change to the Python 2 behaviour.